bin/release

```diff
@@ -60,34 +64,61 @@ function getGit() {
 /**
  * Generates the filename to use for the resulting release zip file.
  *
- * @since 1.0.0
+ * @since n.e.x.t
  *
  * @return {string} Sanitized release zip file name.
  */
 function generateFilename() {
 	const version = getPluginVersion();
-
 	let gitSuffix = '';
+
 	try {
 		const { branch, shortSha } = getGit();
 		gitSuffix = `.${ branch }@${ shortSha }`;
-	} catch {}
+	} catch ( err ) {
+		// Do nothing.
+	}
 
-	return sanitizeFilename(
-		`google-site-kit.v${ version }${ gitSuffix }.zip`
-	);
+	return sanitizeFilename( `google-site-kit.v${ version }${ gitSuffix }.zip` );
 }
 
-module.exports = function() {
+/**
+ * Makes a new release.
+ *
+ * @since n.e.x.t
+ *
+ * @return {Promise} A promise instance.
+ */
+async function makeRelease() {
+	const root = dirname( __dirname );
+	const releaseDir = resolve( root, 'release' );
+	const pluginDir = resolve( releaseDir, 'google-site-kit' );
 	const filename = generateFilename();
 
-	// Remove any existing file by the same name.
-	del.sync( path.resolve( __dirname, `../${ filename }` ) );
+	const cp = ( what ) => fse.copy( resolve( root, what ), resolve( pluginDir, what ) );
+
+	// Remove and re-create release directory.
+	await fse.remove( releaseDir );
+	await fse.ensureDir( pluginDir );
 
-	// eslint-disable-next-line no-console
-	console.log( `Creating ${ filename }` );
```

Comment:
I think this would be good to keep since the name is dynamic.

Reply:
Added back.